src/components/login-view/login-page.jsx

```diff
@@ -12,51 +15,77 @@ export const LoginPage = ({ onLoggedIn }) => {
    const handleSubmit = (e) => {
       e.preventDefault();
       const data = {
-         username: userName,
-         password: pass
+         Name: userName,
+         Password: pass
       };
 
-      fetch("https://movie-api-uahq.onrender.com/login", {
+      fetch("https://movie-api-wbl0.onrender.com/login", {
          method: "POST",
          headers: { "Content-Type": "application/json" },
          body: JSON.stringify(data)
-      }).then((response) => response.json())
+      })
+         .then((response) => {
+            if (!response.ok) {
+               throw new Error("Network response was not ok");
+            }
+            return response.json();
+         })
          .then((data) => {
-            console.log(data);
             if (data.user) {
                localStorage.setItem("user", JSON.stringify(data.user.Name));
                localStorage.setItem("token", data.token);
                onLoggedIn(data.user.Name);
-               // onLoginSubmit(data.user.Name, data.token);
-            }
-            else {
-               alert("Login failed");
+            } else {
```

Comment:
As mention in A. `onLoggedIn` was defined with two parameters. Here you're calling it with a single argument. And it's not a object.
Refactor; `onLoggedIn(data.user, data.token);`

Reply:
+